src/components/implementation/tests/micro_booter/micro_booter.c

```diff
@@ -56,6 +58,50 @@ static void
 tls_set(size_t off, unsigned long val)
 { __asm__ __volatile__("movl %0, %%gs:(%1)" : : "r" (val), "r" (off) : "memory"); }
 
+long long total_swt_cycles = 0LL;
```

Comment:
The code should only reflect the current state of the code.  We won't remember different "possibilities" in a month.